backend/api/index.js

```diff
@@ -35,6 +36,20 @@ router.post(
   })
 );
 
+router.get(
+  '/validateLink/:channel',
+  asyncHandler(async (req, res) => {
+    const { channel } = req.params;
+    const isChannelValid = await channelValid(channel);
+
+    if (!isChannelValid) {
+      return res.sendStatus(404).send('Invalid channel');
+    }
+
+    return res.sendStatus(200).send('Valid channel');
```

Comment:
For any 2.x.x we are expecting a json response. Also for consistency let's just make it 
`{ status: 'ok' }`